src/pvrclient-nextpvr.h

```diff
@@ -201,6 +201,8 @@ class cPVRClientNextPVR : P8PLATFORM::CThread
   std::map<std::string, std::string> m_hostFilenames;
   std::map<int, bool> m_channelTypes;  // returns isRadio
   std::map<int, std::string> m_liveStreams;
+  std::map<std::string,int> m_epgOidLookup;
```

Comment:
Just a general comment but the use of spacing just after commas appears in consistent with the existing code (such as the line above this). It catches my eye each time I see your code ;)

Reply:
I use a mix of linux and windows building and testing this code, so perhaps the autospacing is different in vs code or visual stdio I will watch for it, this doesn't bother me but I will resubmit it.